### PR TITLE
docs: adds section on user reset [DHIS2-9820]

### PR DESCRIPTION
**Pull request by jbee**
Documentation for changes in https://github.com/dhis2/dhis2-core/pull/7725

### CODE REVIEW

File: developer/web-api/users.md [hunk 1]
```diff
@@ -333,6 +333,16 @@ curl -d @replica.json "localhost/api/33/users/N3PZBUlN8vq/replica"
   -H "Content-Type:application/json" -u admin:district
 ```
 
+### Reset user password by invitation { #webapi_user_reset }
+
+User administrators (with appropriate rights) can reset another user's password 
+through a link send to their email. Users following the link get to a form
```
Comment (larshelge) [inline developer/web-api/users.md]: "Sent" instead of "send". Maybe split this into two sentences.

Comment (jbee) [reply developer/web-api/users.md]: Given the change to password recovery I rephrased the whole paragraph.

File: developer/web-api/users.md [hunk 2]
```diff
@@ -333,6 +333,17 @@ curl -d @replica.json "localhost/api/33/users/N3PZBUlN8vq/replica"
   -H "Content-Type:application/json" -u admin:district
 ```
 
+### Reset user password { #webapi_user_reset }
+
+User administrators (with appropriate rights) can reset another user's account
+by triggering password recovery. Once triggered an email is send to the user
```
Comment (larshelge) [inline developer/web-api/users.md]: Use "sent" instead of "send".

Comment (jbee) [reply developer/web-api/users.md]: You are right - I had to look this up thinking in combination with _is_ it should be _send_ but it is still _sent_. :+1:

**Review by larshelge — APPROVED** [review]

**Pull request merged**